docs/protocol/src/shielded_pool/note_plaintexts.md

```diff
@@ -5,6 +5,6 @@ Plaintext notes contain:
 * the value to be transmitted which consists of an integer amount $v$ along with a scalar (32 bytes) $ID$ identifying the asset.
 * $rseed$, a 32-byte random value, which will later be used to derive the note blinding factor used for the
 note commitment and an ephemeral secret key.
-* the destination address, described in more detail in the [Addresses](../addresses_keys/addresses.md) section.
+* the destination address controlling the note, described in more detail in the [Addresses](../addresses_keys/addresses.md) section.
```

Comment:
In the note plaintext [section](https://hackmd.io/v0qMQRKqSz-OK0G4Ml383Q#Note-plaintexts) of the audit, there was some confusion about `pk_d`. Comparing the [spec](https://protocol.penumbra.zone/main/shielded_pool/note_plaintexts.html) against the code reveals some incongruences:

- the `Address` struct defines `pk_d` as the public key used for the payment address, but the spec defines `pk_d` 
as the diversified transmission key. 
- the `Note` struct consists of value, rseed, address, and transmission_key but the [spec](https://protocol.penumbra.zone/main/shielded_pool/note_plaintexts.html) is missing the diversified transmission key.